File: 1116.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
let n = Number(lines.shift())
for(var i = 0; i<n; i++){
    let [a, b] = lines.shift().split(' ')
    if(isFinite(parseFloat(a) / parseFloat(b)) === false){
        console.log('divisao impossivel')
    }    
    else{    
        console.log(`${(parseFloat(a)/parseFloat(b)).toFixed(1)}`)
    }    
}

File: 1116.java
import java.util.Scanner;
import java.util.Locale;
public class Main {
public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        for(int i = 0; i < n; i++){
                int a = input.nextInt(); int b = input.nextInt();
                if(b == 0){
                        System.out.println("divisao impossivel");
                }
                else{
                        double f = (double)a/b;
                        System.out.printf("%.1f%n", f);
                }
        }
}
}
